src/main/java/br/com/vaasschool/model/Subcategory.java

```diff
@@ -30,6 +31,24 @@ public Subcategory(String name, String code, String description, Boolean active,
         this.category = category;
     }
 
+    public Subcategory(int id, String name, String code, String description, Boolean active, Integer order, Category category) {
+        Validator.notNullOrEmpty(name);
+        Validator.isCode(code);
+        Validator.notNull(category);
+
+        this.id = id;
+        this.name = name;
+        this.code = code;
+        this.description = description;
+        this.active = active;
+        this.order = order;
+        this.category = category;
+    }
```

Comment:
Ana ao invés de ter criado o construtor dessa forma você poderia ter chamado o de cima usando o `this`

então ficaria this(name, code,description,active,order,category)

Reply:
Perfeito! Recomendo essa mudança!